app/app.go

```diff
@@ -938,12 +937,12 @@ func (app *App) RegisterTendermintService(clientCtx client.Context) {
 
 // RegisterSwaggerAPI registers swagger route with API Server
 func RegisterSwaggerAPI(ctx client.Context, rtr *mux.Router) {
-	statikFS, err := fs.NewWithNamespace("cronos")
+	root, err := fs.Sub(docs.SwaggerUI, "swagger-ui")
```

Comment:
we were using namespace because of a conflicts issue with the registration in sdk, I don't know if we need it here as well.